server/src/main/resources/server-reference.conf

```diff
@@ -18,6 +18,9 @@ server.electronPath = ""
 server.downloadAsCbz = false
 server.downloadsPath = ""
 
+# updater
+server.maxParallelUpdateRequests = 10 # sets how many sources can be updated in parallel. updates are grouped by source and all mangas of a source are updated synchronously
```

Comment:
Tachiyomi sets this to 5, what can a good default be? Why 10?

Reply:
I saw that as well, my guess was that it's because their running on smartphones.
I thought that this was not the case for the Tachidesk-Server so I increased it a bit, so there is no real reason for the 10, just that I thought 5 was a little too low.

I also don't really now what the average use case is, meaning how many different sources you have in your library.
E.g. I have only 5 different sources atm so the max parallel update requests would be 5 anyway.

But since it is now included in the config, I don't really care what the default value is, so if you or someone else prefers something else it can be changed